app/controllers/questions_controller.rb

```diff
@@ -1,2 +1,26 @@
 class QuestionsController < ApplicationController
-end
+    before_action :set_quiz
+  
+    def new
+      @question = @quiz.questions.build
+    end
+  
+    def create
+      @question = @quiz.questions.build(question_params)
+      if @question.save
+        redirect_to quiz_path(@quiz)
+      else
+        render :new
+      end
+    end
+  
+    private
+  
+    def set_quiz
```

Comment:
optional and a bit of a nit. Usually in Rails apps people will use `find_x` rather than `set`. Just because it's descriptive of the fact it's using the `.find` method to grab a record from the db